catalyst_voices/lib/configs/bootstrap.dart

```diff
@@ -1,35 +1,71 @@
 import 'dart:async';
 import 'dart:developer';
 
+import 'package:catalyst_voices/app/app.dart';
 import 'package:catalyst_voices/configs/app_bloc_observer.dart';
 import 'package:catalyst_voices/configs/sentry_service.dart';
+import 'package:catalyst_voices/dependency/dependencies.dart';
+import 'package:catalyst_voices/routes/guards/milestone_guard.dart';
+import 'package:catalyst_voices/routes/routes.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:go_router/go_router.dart';
 import 'package:url_strategy/url_strategy.dart';
 
-Future<void> bootstrap(FutureOr<Widget> Function() builder) async {
+typedef BootstrapWidgetBuilder = FutureOr<Widget> Function(BootstrapArgs args);
+
+final class BootstrapArgs {
+  BootstrapArgs({
+    required this.routerConfig,
+  });
+
+  final RouterConfig<Object> routerConfig;
+}
+
+Widget _appBuilder(BootstrapArgs args) {
```

Comment:
Consider to put _appBuilder below bootstrap so that code can be ready top-to-bottom